--- a/src/asl/CMakeLists.txt	2019-02-08 11:04:56.961620700 -0600
+++ b/src/asl/CMakeLists.txt	2019-02-08 11:05:44.709402200 -0600
@@ -264,10 +264,7 @@ if (NOT HAVE_GETRUSAGE)
   target_compile_definitions(asl-extra PRIVATE NO_RUSAGE)
 endif ()
 
-find_library(DL_LIBRARY dl)
-if (DL_LIBRARY)
-  target_link_libraries(asl dl)
-endif ()
+target_link_libraries(asl ${CMAKE_DL_LIBS})
 
 find_package(MATLAB)
 if (MATLAB_FOUND)
