--- a/SRC/dsytrf_aa_2stage.f	Mon Nov 13 05:15:54 2017
+++ b/SRC/dsytrf_aa_2stage.f	Wed Jan 03 15:41:24 2018
@@ -191,7 +191,7 @@
       EXTERNAL           LSAME, ILAENV
 *     ..
 *     .. External Subroutines ..
-      EXTERNAL           XERBLA, DCOPY, DLACGV, DLACPY,
+      EXTERNAL           XERBLA, DCOPY, DLACPY,
      $                   DLASET, DGBTRF, DGEMM,  DGETRF, 
      $                   DSYGST, DSWAP, DTRSM 
 *     ..
--- a/SRC/ssytrf_aa_2stage.f	Mon Nov 13 05:15:54 2017
+++ b/SRC/ssytrf_aa_2stage.f	Wed Jan 03 15:40:18 2018
@@ -191,7 +191,7 @@
       EXTERNAL           LSAME, ILAENV
 *     ..
 *     .. External Subroutines ..
-      EXTERNAL           XERBLA, SCOPY, SLACGV, SLACPY,
+      EXTERNAL           XERBLA, SCOPY, SLACPY,
      $                   SLASET, SGBTRF, SGEMM,  SGETRF, 
      $                   SSYGST, SSWAP, STRSM 
 *     ..
