--- a/makefile	Mon Jul 02 17:58:05 2018
+++ b/makefile	Wed Aug 01 10:56:27 2018
@@ -95,28 +95,28 @@
 	-@echo "Using configuration flags:"
 	-@grep "\#define " ${PETSCCONF_H}
 	-@echo "-----------------------------------------"
-	-@echo "Using C/C++ compile: ${PETSC_COMPILE}"
-	-@if [ "${PETSC_LANGUAGE}" = "CONLY" -a "${MPICC_SHOW}" != "" ]; then \
-             printf  "mpicc -show: %b\n" "${MPICC_SHOW}"; \
-	  elif [ "${PETSC_LANGUAGE}" = "CXXONLY" -a "${MPICXX_SHOW}" != "" ]; then \
-             printf "mpicxx -show: %b\n" "${MPICXX_SHOW}"; \
-          fi;
-	-@if [ "${FC}" != "" ]; then \
-	   echo "Using Fortran compile: ${PETSC_FCOMPILE}";\
-           if [ "${MPIFC_SHOW}" != "" ]; then \
-             printf "mpif90 -show: %b\n" "${MPIFC_SHOW}"; \
-           fi; \
-         fi
-	-@if [ "${CUDAC}" != "" ]; then \
-	   echo "Using CUDA compile: ${PETSC_CUCOMPILE}";\
-         fi
-	-@echo "-----------------------------------------"
-	-@echo "Using C/C++ linker: ${PCC_LINKER}"
-	-@echo "Using C/C++ flags: ${PCC_LINKER_FLAGS}"
-	-@if [ "${FC}" != "" ]; then \
-	   echo "Using Fortran linker: ${FC_LINKER}";\
-	   echo "Using Fortran flags: ${FC_LINKER_FLAGS}";\
-         fi
+	# -@echo "Using C/C++ compile: ${PETSC_COMPILE}"
+	# -@if [ "${PETSC_LANGUAGE}" = "CONLY" -a "${MPICC_SHOW}" != "" ]; then \
+    #          printf  "mpicc -show: %b\n" "${MPICC_SHOW}"; \
+	#   elif [ "${PETSC_LANGUAGE}" = "CXXONLY" -a "${MPICXX_SHOW}" != "" ]; then \
+    #          printf "mpicxx -show: %b\n" "${MPICXX_SHOW}"; \
+    #       fi;
+	# -@if [ "${FC}" != "" ]; then \
+	#    echo "Using Fortran compile: ${PETSC_FCOMPILE}";\
+    #        if [ "${MPIFC_SHOW}" != "" ]; then \
+    #          printf "mpif90 -show: %b\n" "${MPIFC_SHOW}"; \
+    #        fi; \
+    #      fi
+	# -@if [ "${CUDAC}" != "" ]; then \
+	#    echo "Using CUDA compile: ${PETSC_CUCOMPILE}";\
+    #      fi
+	# -@echo "-----------------------------------------"
+	# -@echo "Using C/C++ linker: ${PCC_LINKER}"
+	# -@echo "Using C/C++ flags: ${PCC_LINKER_FLAGS}"
+	# -@if [ "${FC}" != "" ]; then \
+	#    echo "Using Fortran linker: ${FC_LINKER}";\
+	#    echo "Using Fortran flags: ${FC_LINKER_FLAGS}";\
+    #      fi
 	-@echo "-----------------------------------------"
 	-@echo "Using system modules: ${LOADEDMODULES}"
 	-@TESTDIR=`mktemp -q -d -t petscmpi-XXXXXXXX` && \
