diff --git a/cmake/Modules/FindFLANN.cmake b/cmake/Modules/FindFLANN.cmake
index b5739dc95..680c481f5 100644
--- a/cmake/Modules/FindFLANN.cmake
+++ b/cmake/Modules/FindFLANN.cmake
@@ -9,13 +9,15 @@
 # If FLANN_USE_STATIC is specified and then look for static libraries ONLY else
 # look for shared ones
 
-if(FLANN_USE_STATIC)
-  set(FLANN_RELEASE_NAME flann_cpp_s)
-  set(FLANN_DEBUG_NAME flann_cpp_s-gd)
-else(FLANN_USE_STATIC)
-  set(FLANN_RELEASE_NAME flann_cpp)
-  set(FLANN_DEBUG_NAME flann_cpp-gd)
-endif(FLANN_USE_STATIC)
+#if(FLANN_USE_STATIC)
+#  set(FLANN_RELEASE_NAME flann_cpp_s)
+#  set(FLANN_DEBUG_NAME flann_cpp_s-gd)
+#else(FLANN_USE_STATIC)
+#  set(FLANN_RELEASE_NAME flann_cpp)
+#  set(FLANN_DEBUG_NAME flann_cpp-gd)
+#endif(FLANN_USE_STATIC)
+set(FLANN_RELEASE_NAME flann)
+set(FLANN_DEBUG_NAME flann)
 
 find_package(PkgConfig QUIET)
 if (FLANN_FIND_VERSION)
